src/lib/HomeStore.js

```diff
@@ -106,7 +106,8 @@ export default class HomeStore {
     const normalizedJob = normalize(job, JOBS_DOCTYPE)
     // TODO Filter by worker on the WebSocket when it will be available in the
     // stack
-    const isKonnectorJob = normalizedJob.worker === 'konnector'
+    const isKonnectorJob =
```

Comment:
can't we use https://github.com/cozy/cozy-home/pull/1598/files#diff-45605170e214db04efacafd86775c0e2db2155289736be4553aa8b4851ba9b2aR33-R35 ? 

This kind of method should be put in Cozy-Client.

Reply:
Yes, done (not in cozy-client yet)